### PR TITLE
[#7] UI Menu Component

### PR DESCRIPTION
**Pull request by GoraJakub**
Please review. I believe there's many mistakes to fix.

### CODE REVIEW

File: src/app/components/mainMenu.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        throw new Error("Callback argument ins't a function!");
+    }
+
+    //Created nav container
+    const menuContainer = document.createElement("nav");
+    menuContainer.classList.add("swquiz-menu");
+    menuContainer.id = 'menuContainer';
+    menuContainer.setAttribute("data-testid","menuContainer")
+    //Created button array
+    const buttons = [
```
Comment (mfrydrychowicz) [inline src/app/components/mainMenu.js:15]: I think you may split this function into smaller parts
eg.
createButtons () {
// creatung buttons
return buttons
}

File: test/mainMenu.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import {mainMenu} from '../src/app/components/mainMenu';
```
Comment (mfrydrychowicz) [inline test/mainMenu.test.js:1]: please run `npm run format`

File: src/app/App.js [hunk 3]
```diff
@@ -1,4 +1,12 @@
+import {mainMenu} from "./components/mainMenu.js";
+
 export const App = ({options}) => {
+    const getData = (element) =>{
+        return element;
+    }
+    mainMenu(document.body,getData);
```
Comment (mfrydrychowicz) [inline src/app/App.js:7]: Shouldn't it be : 
    `mainMenu(document.querySelector('.swquiz-header'),getData);`

and then: 
remove `flex-wrap: wrap;` from style .swquiz-header in file gameHeader.css beacuse it render wrong right now (my bad!)

**Review by mfrydrychowicz — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz#readme",
   "dependencies": {
+    "components": "^0.1.0",
```
[the remark this answers is not in the document]
Comment (budnickip) [reply package.json:28]: i agree with @mfrydrychowicz

File: src/app/components/mainMenu.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    throw new Error("Callback argument isn't a function!");
+  }
+
+  //Created nav container
+  const menuContainer = document.createElement('nav');
+  menuContainer.classList.add('swquiz-menu');
+  menuContainer.id = 'menuContainer';
+  menuContainer.setAttribute('data-testid', 'menuContainer');
+
+  const btnPeople = createButton('swquiz-menu-button', 'People', callback);
```
Comment (mfrydrychowicz) [inline src/app/components/mainMenu.js:30]: could you import variables from modes.js and use them instead of hardcoding? That would be pro 😃

**Review by mfrydrychowicz — CHANGES_REQUESTED** [review]

**Review by budnickip — APPROVED** [review]
Like Małgorzata says, if you have time, you can import consts from modes.js

**Review by mfrydrychowicz — DISMISSED** [review]

**Review by budnickip — APPROVED** [review]

**Pull request merged**